tests/jvm/src/test/scala/scala/meta/tests/semanticdb/ExpectSuite.scala

```diff
@@ -25,8 +25,8 @@ class ExpectSuite extends FunSuite with DiffAssertions {
     // output of 2.12. It's possible to add another expect file for 2.12
     // later down the road if that turns out to be useful.
     case "2" :: "12" :: Nil =>
-      test("convert.expect") {
-        import ConvertExpect._
+      test("metacp.expect") {
```

Comment:
I feel like we should test the diff between metacp and metac for symbols that are covered by metacp. In theory they should produce identical types making the diff empty but I suspect that may not be the case, if there are inconsistencies I want them documented in our tests.

Reply:
Let's do this in a different pull request, since this one is getting pretty big. This is going to be my third rebase, and keeping this PR in the queue for longer will mean more rebases.